client-frameworks-support/client-support-ui5/src/README.md

```diff
@@ -0,0 +1,198 @@
+# Luigi Client UI5 Support Library
+​
+The Luigi Client UI5 Support Library offers a set [features](#features) which make it easier to use the [Luigi micro frontend framework](https://luigi-project.io/) with UI5 applications. 
+​
+## How to use the library
+
+As a first step you need to add `ComponentSupport` to the `index.html` as well as to allow the `data-sap-ui-frameOptions`.
+```javascript
+<script id="sap-ui-bootstrap" src="resources/sap-ui-core.js" data-sap-ui-resourceroots='{
+				"luigi.ui5-demoapp": "./"
+			}' data-sap-ui-oninit="module:sap/ui/core/ComponentSupport" data-sap-ui-compatVersion="edge"
+		data-sap-ui-async="true" data-sap-ui-frameOptions="allow" data-sap-ui-theme="sap_horizon"
+		data-sap-ui-xx-waitForTheme="true" data-sap-ui-xx-supportedLanguages="en,de">
+		</script>
+```
+### Installing via npm
+​
+1. Import the library in your `package.json` file:
+```javascript
+npm install @luigi-project/client-support-ui5
+```
+
+2. If your project is not set up for use with the [UI5 Tooling](https://sap.github.io/ui5-tooling/stable/) yet, you need to install it. It is required to consume the ui5-support-lib.
+​
+3. In the last step you need to register the library in your `Component.js` file.
+
+Using UI5 tooling:
+```javascript
+sap.ui.define([
+	"sap/ui/core/UIComponent",
+    ....
+	"@luigi-project/client-support-ui5/ui5-support-lib",
+], function (UIComponent,...., Ui5SupportLib) {
+    return UIComponent.extend("com.sap.luigiclient.Component", {
+		....
+		init: function () {
+			.....			
+			// connect client-support-ui5 lib to the application
+			Ui5SupportLib.connectTo(this);
+            
+            this.LuigiClient = Ui5SupportLib.LuigiClient;
+​
+			this.getRouter().initialize();
+		},
+```
+### Installing the lib manually
+
+1. Create a `lib` folder under the `webapp` directory and put the `luigi-client.js` and `ui5-support-lib.js` into it.
```

Comment:
```suggestion
### Installing manually

1. Create a `lib` folder under the `webapp` directory and put the [luigi-client.js](../../../client/src/luigi-client.js) and [ui5-support-lib.js](ui5-support-lib.js) files into it.
```
I think it's better to have links to the specific files. (I'm not 100% sure if these are the files you mean though.)

Reply:
Good point 👍 
I changed the link and point it to npmjs because we need the bundled versions.